proof/refine/AARCH64/CNodeInv_R.thy

```diff
@@ -4962,6 +4962,26 @@ lemmas tcbSlots =
   tcbCTableSlot_def tcbVTableSlot_def
   tcbReplySlot_def tcbCallerSlot_def tcbIPCBufferSlot_def
 
+context Arch begin arch_global_naming
+
+lemma updateMDB_pspace_in_kernel_mappings'[wp]:
+  "updateMDB next prev \<lbrace>pspace_in_kernel_mappings'\<rbrace>"
+  by wpsimp
+
+lemma updateCap_pspace_in_kernel_mappings'[wp]:
+  "updateCap next prev \<lbrace>pspace_in_kernel_mappings'\<rbrace>"
+  by wpsimp
+
+end
+
+(* FIXME arch-split: interface (these are only interesting on ARM/ARM_HYP *)
+arch_requalify_facts
+  updateMDB_pspace_in_kernel_mappings'
+  updateCap_pspace_in_kernel_mappings'
```

Comment:
I'm a bit confused by this comment with these requalifies only appearing on `AARCH64`.

Reply:
It's trying to say that this is interesting on ARM/ARM_HYP, but trivial here, hence we crunch+requalify. I guess judging by your reaction this is not very useful...

Reply:
I guess it doesn't really matter, since presumably this comment will go away once arch-split reaches CNodeInv_R. I'm still a bit confused though about them being interface lemmas when they don't appear on `X64` or `RISCV64`, but again that will be resolved later once we reach here.

Reply:
Yeah, I think that's the right move for now. The FIXMEs that appear in downstream  theories usually indicate there's some work to be done that we don't want to forget.